_layouts/wiki-de.html

```diff
@@ -36,13 +36,9 @@
 
         <!--   Core JS Files   -->
         <script src="/assets/js/core/jquery-3.6.0.min.js" type="text/javascript"></script>
-        <script src="/assets/js/core/bootstrap-4.6.1.bundle.min.js" type="text/javascript"></script>
-        <script src="/assets/js/details-element-polyfill.js" type="text/javascript"></script>
+        <script src="/assets/js/core/bootstrap-5.1.3.bundle.min.js" type="text/javascript"></script>
         <script> //open dropdown by default if on Desktop, but not on Mobile
-            const breakpoint = parseInt(
-                getComputedStyle(document.documentElement).getPropertyValue('--breakpoint-xl')
-            );
-            if (screen.width >= breakpoint){
+            if (screen.width >= 1200){
```

Comment:
is there a specific reason 1200 was selected?

Reply:
This is the size of the xl breakpoint for bootstrap https://getbootstrap.com/docs/5.0/layout/breakpoints/. I can add a comment there explaining that

Reply:
Meant to do that earlier, but forgot about it in the sea of things I had to change